### DISCUSSION TITLE
Template

### DISCUSSION THREAD

**Discussion by MatejEvc** [SE-TINF22B2/G3-ApeRepublic#79] [Statusreports]
## ⚡ Quicktakes
* Status: 🟢
* 123

## 🥇 Achievements
* 123

 ## 🗺️ Next Steps
* 123

**Comment by elnaz-gharoon** [comment]
@MatejEvc ist weekly report? Du hast es ja erstelt  #73

**Reply by MatejEvc** [reply]
ist ein Template für die zukünftige Statusreports
